docs/intro.md

```diff
@@ -8,23 +8,26 @@ description: Intro to Rollkit, a modular framework for rollups.
 
 Welcome to Rollkit docs. We’re happy you made it here!
 
-Our mission is to empower developers to quickly innovate and create entire new classes of rollups with minimal tradeoffs.
-
-We’re setting the bar high for developers’ flexibility and ability to customize rollups however they see fit.
-
 :::tip Tip
 If you're familiar with Rollkit, you may want to skip to the [tutorials section](../category/tutorials)
 :::
 
 ## What is Rollkit?
 
-Rollkit is a rollup framework that gives developers the freedom to deploy rollups throughout the modular stack with minimal tradeoffs, opening new possibilities for rapid experimentation and innovation.
+Rollkit is a rollup framework that gives developers the freedom to deploy rollups throughout the modular stack, opening new possibilities for rapid experimentation and innovation.
+
+The Rollkit framework features a modular node that can run rollups and expose an **[ABCI](https://github.com/informalsystems/tendermint/tree/main/abci)**-compatible client interface, which can be used as a substitute for Tendermint in any ABCI-compatible blockchain application.\
+By default, the node utilizes Celestia as the DA layer. In the future, the node will be able to connect to any DA layer of choice.
 
-It was built as a drop-in replacement for Tendermint for any ABCI-compatible blockchain app.
+In addition to deploying sovereign app-rollups, Rollkit can also be used to deploy dapps on existing settlement and execution layers, or even to create a new settlement layer. 
```

Comment:
how can i use rollkit to deploy dapps on existing settlement/execution layers? a dapp is not a rollup

Reply:
Good catch, wanted to say non-sovereign rollups

Reply:
we can maybe just say,  "Rollkit can also be used to deploy rollups on existing settlement and execution layers...

Reply:
oh, so like an L3?

Reply:
exactly

Reply:
wanna make the edit?

Reply:
what about this one?